src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/refdatafixes/RefDataFixesCcdDataSource.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.service.refdatafixes;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.hmcts.ecm.common.client.CcdClient;
+import uk.gov.hmcts.ecm.common.model.ccd.SubmitEvent;
+
+@RequiredArgsConstructor
+@Slf4j
+public class RefDataFixesCcdDataSource implements RefDataFixesDataSource {
+
+    private final String authToken;
+
+    @Override
+    public List<SubmitEvent> getData(String caseTypeId, String dateFrom, String dateTo, CcdClient ccdClient) {
+        try {
+            var query = RefDataFixesElasticSearchQuery.create(
```

Comment:
don't know what we are doing with var in the ethos project :)

Reply:
removed